tox.ini

```
@@ -83,11 +83,13 @@ description = Run integration tests
 deps =
     aiohttp
     git+https://github.com/juju/python-libjuju.git
-    #juju
+    juju
```

On pypi, python-libjuju is just juju. We use it from gh.
```suggestion
    #juju
```